core/network/impl/stream_engine.hpp

```diff
@@ -97,7 +97,7 @@ namespace kagome::network {
       auto &proto_map = syncing_streams_[peer];
       proto_map.emplace(protocol, std::move(stream));
       logger_->debug("Syncing stream (peer_id={}, protocol={}) was emplaced",
-                     peer.id.toHex(),
+                     peer.id.toBase58(),
```

Comment:
Thanks! Seeing hex peer addres in logs was annoying

Reply:
Absolutly